src/ensembl/src/shared/components/external-link/ExternalLink.tsx

```diff
@@ -37,15 +37,20 @@ const ExternalLink = (props: ExternalLinkProps) => {
 
   return (
     <span className={styles.container}>
-      <LinkIcon className={iconClass} />
-      <a
-        className={linkClass}
-        href={props.to}
-        target="_blank"
-        rel="nofollow noreferrer"
-      >
-        {props.linkText}
-      </a>
+      {props.to && (
+        <>
+          <LinkIcon className={iconClass} />
+          <a
+            className={linkClass}
+            href={props.to}
+            target="_blank"
+            rel="nofollow noreferrer"
+          >
+            {props.linkText}
+          </a>
+        </>
+      )}
+      {!props.to && <span className={styles.noLink}>{props.linkText}</span>}
```

Comment:
Will using a ternary operator be better here?

Reply:
This no longer applies here